infra/src/test/kotlin/fi/fta/geoviite/infra/math/GeographyTest.kt

```diff
@@ -46,6 +48,44 @@ class GeographyTest {
         assertApproximatelyEquals(point3857, transformed3857, 0.01)
     }
 
+    @Test
+    fun tm35finToGKWorksInEast() {
```

Comment:
Vanhempien testien nimitykset ovatkin näin yhteen, uudemmissa on suosittu erottamista käyttämällä backtickkejä ja välejä lukemisen helpottamiseksi:

Siis esim. ``fun `TM35FIN to GK works in east`()``